frontend/views/partials/landing.html

```diff
@@ -37,9 +55,16 @@
             <button class="btn btn-lg btn-primary" type="button" ng-click="loginGo()">Login</button>
             &nbsp;
              <button class="btn btn-lg btn-primary"  type="button" ng-click="signupGo()">Sign Up</button>
```

Comment:
@Israa-Elwan 
please add section identifier to know what begins where
like
<!-edit form starts here-->

Html code
<!-edit form ends here-->